### PR TITLE
Fix actions dropdown

### PR DESCRIPTION
**Pull request by wfortin**
Remove complexity of the dropdown.
Fixed padding when open and closed to prevent glitches.
Fixed activity.svg

image
image

### CODE REVIEW

File: resources/icons/svg/activity.svg [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-    <circle fill-rule="evenodd" clip-rule="evenodd" cx="-264.367" cy="421.89" r="1.6"/>
-	<path d="M-253.033,432.89h-7.339c-0.552,0-1-0.448-1-1s0.448-1,1-1h7.339c0.552,0,1,0.448,1,1
-		S-252.48,432.89-253.033,432.89z"/>
-	<path d="M-253.033,427.89h-7.339c-0.552,0-1-0.448-1-1s0.448-1,1-1h7.339c0.552,0,1,0.448,1,1
-		S-252.48,427.89-253.033,427.89z"/>
-	<path d="M-253.033,422.89h-7.339c-0.552,0-1-0.448-1-1s0.448-1,1-1h7.339c0.552,0,1,0.448,1,1
-		S-252.48,422.89-253.033,422.89z"/>	
+<svg width="20" height="20" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" viewBox="0 0 20 20" >
+ <g>
+  <title>Layer 1</title>
```
Comment (JPLachance) [inline resources/icons/svg/activity.svg]: The title tag is useless.

**Comment by JPLachance** [comment]
Look's good!

**Comment by GermainBergeron** [comment]
:shipit:

File: resources/icons/svg/activity.svg (hunk 1 above)
Comment (wfortin) [inline resources/icons/svg/activity.svg]: svg editor adds it back when i remove it. So i guess it's not optional ?

File: resources/icons/svg/activity.svg [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+  <g id="svg_1" stroke="null">
```
Comment (pochretien) [inline resources/icons/svg/activity.svg]: remove id to avoid collision with other svg

**Comment by pochretien** [comment]
Very nice clean up !! ship it

File: resources/icons/svg/activity.svg (hunk 1 above)
Comment (JPLachance) [inline resources/icons/svg/activity.svg]: Hum, remove it in a text editor, render the svg, the result is the same => probably useless ;)

Comment (wfortin) [inline resources/icons/svg/activity.svg]: cleaned

File: resources/icons/svg/activity.svg (hunk 2 above)
Comment (wfortin) [inline resources/icons/svg/activity.svg]: removed.

**Pull request merged**